modules/standard/IO.chpl

```diff
@@ -8740,9 +8733,9 @@ proc _channel.skipField() throws {
   :arg args: the arguments to format
   :returns: the resulting string
 
-  :throws EofError
-  :throws UnexpectedEofError
-  :throws BadFormatError
+  :throws EofError: EOF encountered.
+  :throws UnexpectedEofError: EOF encountered before all data could be read or written.
```

Comment:
For these two format methods, I'd probably remove `EofError` from the docs and then change the description of `UnexpectedEofError` to "The size of the temporary buffer was exceeded while writing the string/bytes"